.eslintrc.json

```diff
@@ -13,6 +13,7 @@
   "rules": {
     "import/prefer-default-export": "off",
     "no-shadow": "off",
+    "max-classes-per-file": "off",
```

Comment:
- [ ] I would recommend to not change the `.eslintrc.json` file. As we have to follow the standard and best practices. Please use the recommended `eslintrc` file from [here](https://github.com/microverseinc/linters-config/blob/master/html-css-js/.eslintrc.json).